cli/src/common/utils.py

```diff
@@ -43,3 +46,26 @@ def validate_json_format(data):
     except ValueError:
         return False
     return True
+
+
+def get_url_scheme(url):
+    """Parse url to get scheme."""
+    return urlparse(url).scheme
+
+
+def get_bucket_object_name(url):
+    """
+    Parse s3 url to get bucket name and object name.
+
+    input: s3://test/templates/3.0/post_install.sh
+    output: {"bucket_name": "test","full_object_name": "templates/3.0/post_install.sh","object_name": "post_install.sh"}
+    """
+    match = re.match(r"s3://(.*?)/(.*)", url)
+    if match:
+        bucket_name = match.group(1)
+        full_object_name = match.group(2)
```

Comment:
minor - we can be consistent with name convention.
the `full_object_name` is called `object_key`, see https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingObjects.html
then we can call `object_name` to be `file_name`

Reply:
done